### ISSUE TITLE
Details about resuming training

### ISSUE THREAD

**Issue by escorciav** [facebookresearch/SlowFast#121]
Hi!

Thanks, for releasing this and for structuring the code in such a nice way.

_Could you please provide details about what's done when resuming training from an existing checkpoint? In particular,_

1. Are gradients history loaded as well?

BTW, it's super cool that the epochs are shifted in the LOG!

Cheers,
Victor

**Comment by takatosp1** [comment]
Hi @escorciav 

Thanks a lot for playing with PySlowFast! Also feel glad you like the shifted epochs :P 

There are generally 3 different cases for loading checkpiont

1) resume a training job, 2) load and inflated a 2D pretrain model, and 3) load a trained model.

The grad is not loaded but for 1) the stats in optim is loaded.

Feel free to share your comments and I am very curious about your use case.

**Comment by escorciav** [comment]
Thanks, the current setup is enough for me.
